src/Pages/PlayList-Page/Components/Header.css

```diff
@@ -0,0 +1,19 @@
+.header--container {
```

Comment:
You don't have to import header.css on every component you can directly import once on App/Route Components.

Reply:
Okay,ThankYou for Review.